scripts/ubuntu-jammy-from-community-gui/50-wayland_errrata.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+set -exv
+
+# init helpers
+helpers_dir=${MONOPACKER_HELPERS_DIR:-"/etc/monopacker/scripts"}
+for h in ${helpers_dir}/*.sh; do
+    . $h;
+done
+
+# see https://github.com/mozilla-platform-ops/monopacker/issues/138
+
+
+#
+# install tools
+#
+
+# used to modify specific blocks in .conf files
+apt install -y crudini
+
+
+#
+# dconf settings
+#
+
+cat > /etc/dconf/profile/user << EOF
+user-db:user
+system-db:local
+EOF
+
+mkdir /etc/dconf/db/local.d/
+# dconf user settings
+cat > /etc/dconf/db/local.d/00-tc-gnome-settings << EOF
+# /org/gnome/desktop/session/idle-delay
+[org/gnome/desktop/session]
+idle-delay=uint32 0
+
+# /org/gnome/desktop/lockdown/disable-lock-screen
+[org/gnome/desktop/lockdown]
+disable-lock-screen=true
+EOF
+
+# make dbus read the new configuration
+sudo dconf update
+
+# test
+ls -hal /etc/dconf/db/
+
+
+#
+# gdm3 settings  
+#
+
+# in [daemon] block of /etc/gdm3/custom.conf we need:
+#
+# XorgEnable=false
+
+crudini  --set /etc/gdm3/custom.conf daemon XorgEnable 'false'
+
+# verify/test
+cat /etc/gdm3/custom.conf
+echo "----"
+grep 'XorgEnable' /etc/gdm3/custom.conf
+grep 'XorgEnable' /etc/gdm3/custom.conf | grep false
+
+
+#
+# gdm wait service file
```

Comment:
We might want to explain that this is a hack required because without, on those workers, we end up in a situation where the wayland seat is in a weird state and consequences are:
 - either x11 session
 - either xwayland fallback
 - either wayland but with missing keyboard capability that breaks things including copy/paste

Reply:
Thanks. I was unclear about why it was needed. Comment added.